blog.njk

```diff
@@ -0,0 +1,31 @@
+---json
+{
+  "eleventyExcludeFromCollections": true,
+  "permalink": "/blog.xml",
+  "metadata": {
+    "title": "WPE WebKit Blog",
+    "description": "News related to WPE WebKit.",
+    "url": "https://wpewebkit.org/blog/",
+    "feedUrl": "https://wpewebkit.org/blog.xml"
+  }
+}
+---
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's a bit suboptimal that this is basically a copy of the contents of the existing `feed.njk` file, but as I don't know from the top of my head how to avoid the duplication, I can live with this 🙃

Reply:
I filed #112 to see if there's some way of avoiding the duplication later on.

Reply:
Good point. Thanks.